src/main/kotlin/org/rust/cargo/toolchain/Cargo.kt

```diff
@@ -45,6 +45,16 @@ class Cargo(
     private val cargoExecutable: Path,
     private val rustExecutable: Path
 ) {
+    val supportsBuildCheckAllTargets: Boolean = checkSupportForBuildCheckAllTargets()
+
+    private fun checkSupportForBuildCheckAllTargets(): Boolean {
+        val lines = GeneralCommandLine(cargoExecutable)
```

Comment:
I think we can use `CargoCommandLine` here
It will require some `Disposable` to execute but we have it in all places where we need info about `--all-targets` flag

Reply:
Not only that, we would also have to pass the project directory around to create `CargoCommandLine`.  Is it worth it?

Reply:
Hm, maybe not
Then let's move `GeneralCommandLine.runExecutable` into `Utils` and make it public

Reply:
The `Utils` in `org.rust.cargo.runconfig`?

Reply:
in existing `org.rust.cargo.runconfig.Utils`